src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -214,18 +214,20 @@ export class Accept extends Component<AcceptProps, AcceptState> {
               <Flex flexDirection="column">
                 <Flex flexDirection="column">
                   <Media greaterThan="xs">
-                    <ArtworkSummaryItem order={order} />
+                    {className => (
+                      <ArtworkSummaryItem className={className} order={order} />
+                    )}
```

Comment:
Out of curiosity, why was `className` needed here?

Reply:
The `ArtworkSummaryItem` is based on the `StackableBorderBox` which needs to have other `StackableBorderBox`es as siblings in order for the interstitial borders to work correctly. `Media` adds an extra div around the artwork summary by default, which breaks the borders. So we needed to use the render props `Media` pattern to avoid that extra div layer and pass the className directly to the `StackableBorderBox`

Reply:
word 👍 ty